spec/std/yaml/yaml_spec.cr

```diff
@@ -8,7 +8,7 @@ describe "YAML" do
     it { YAML.parse_all("---\nfoo\n---\nbar\n").should eq(["foo", "bar"]) }
     it { YAML.parse("foo: bar").should eq({"foo" => "bar"}) }
     it { YAML.parse("--- []\n").should eq([] of YAML::Type) }
-    it { YAML.parse("---\n...").should be_nil }
+    it { YAML.parse("---\n...").should eq nil }
```

Comment:
can you explain why it should eq nil and not be nil?

Reply:
`YAML.parse` returns a `YAML::Any`, thus it is not `nil` but `any == nil` still returns `true` if `any.value` is `nil`.